common-app-lib/src/cmr/common_app/api/launchpad_token_validation.clj

```diff
@@ -1,17 +1,66 @@
 (ns cmr.common-app.api.launchpad-token-validation
   "Validate Launchpad token."
   (:require
+   [cheshire.core :as json]
+   [clojure.data.codec.base64 :as base64]
+   [clojure.string :as string]
    [cmr.common-app.config :as config]
    [cmr.common.services.errors :as errors]
    [cmr.transmit.config :as transmit-config]))
 
+;; Note: Similar code exists at gov.nasa.echo.kernel.service.authentication
 (def URS_TOKEN_MAX_LENGTH 100)
 
+(defn is-heritage-token?
+  "Heritage token, meaning not the original style legacy token, but a modern
+   token made to be identifiable but also comply with the legacy standard. These
+   tokens are prefixed with 'EDL+'.
+   Note: Similar code exists at gov.nasa.echo.kernel.service.authentication."
+  [token]
+  (string/starts-with? token "EDL+"))
+
+(defn is-legacy-token?
```

Comment:
The whole purpose of this function is to check the length of the token against a certain token length to rule out ECHO tokens and old EDL tokens and help us identify a Launchpad token. I would update the comment to say just that. And we should remove the check of `(is-heritage-token? token)` from the function. I don't see value in that check.

Reply:
I was going to argue for keeping it, but I took another look at the code and will combine two of the functions.